src/components/general/Slider/RangeSlider.tsx

```diff
@@ -120,7 +116,12 @@ const Slider: React.FC<SliderProps> = ({ values, markers, disabled, hideHandle,
                 className={classNames(styles.slider, styles.active)}
                 style={{
                     left: calculatePosition(lastValue > firstValue ? firstValue : lastValue),
-                    width: diffWidth,
+                    right: `${
+                        100 -
+                        +calculatePosition(lastValue > firstValue ? lastValue : firstValue).split(
```

Comment:
Would be better and more readable to use `parseInt(calculatePosition(...), 10)` instead of `+calculatePosition(...).split('%')[0]` i guess. Either that or change calculatePosition to return a number and append the '%' where needet (e.g. line 118)

Reply:
Did the first one, if i did the second one I've have to append % allot of places